publication-constants/README.md

```diff
@@ -0,0 +1,3 @@
+# publication-constants
+
+Contains a database constants that are used in several modules.
```

Comment:
Again, not your fault or relevant, but couldn't this simply be moved to `publication-commons`?  :-(

Reply:
I think the major issue is that publication-commons contains far too much. Would we prefer smaller, specific commons modules that are truly common to all modules that use them (i.e. no "used by two, imported by ten")? There is a naming issue "publication-constants" --> "database-constants".

Reply:
Good point, but this module is only used in `publication-commons` and `publication-testing`, so it doesn't actually solve that problem I think.

Reply:
See task NP-48338